g11n-ws/modules/md-service-i18n-l2/src/main/java/com/vmware/i18n/l2/dao/image/CountryFlagDaoImpl.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright 2019-2023 VMware, Inc.
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package com.vmware.i18n.l2.dao.image;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.vmware.vip.common.constants.ConstantsChar;
+import com.vmware.vip.common.constants.ConstantsFile;
+import com.vmware.vip.common.constants.ConstantsKeys;
+import com.vmware.vip.common.i18n.dto.response.APIResponseDTO;
+import com.vmware.vip.common.i18n.status.APIResponseStatus;
+import jakarta.annotation.PostConstruct;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.support.PathMatchingResourcePatternResolver;
+import org.springframework.stereotype.Repository;
+import org.springframework.util.ResourceUtils;
+
+import java.io.*;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+@Repository
+public class CountryFlagDaoImpl implements ICountryFlagDao {
+    private static Logger logger = LoggerFactory.getLogger(CountryFlagDaoImpl.class);
+    private static String basePath = ConstantsKeys.IMAGE + File.separator + ConstantsKeys.FLAGS + File.separator;
+
+    @PostConstruct
```

Comment:
This convertion executed at runtime, right? I have same comment as @linr211, please move these code to g11n-cldr-pattern branch as other l2 patterns be converted before pattern build publish.

Reply:
It doen't cldr data, so that we can't move data to cldr branch